openhub/outside_projects.py

```diff
@@ -5,8 +5,19 @@
 
 
 def get_outside_projects_data(json_object):
+    """
+    :param json_object: json data of outside projects
+    :return: a list of outside projects dict
+    """
     data = json_object['response']['result'
                                    ]['outside_projects']['project']
+    # When there is only one item on the OpenHub API
+    # page, the return type of the data is 'dict', which
+    # should be converted into the 'list'
+    if isinstance(data, dict):
+        data = [data]
+        # Check if the data contains only one item
+        assert len(data) == 1
```

Comment:
this is silly.  you just created `data` above with a single item.

make assertions on the real `data` you were given, before you assume it is usable.